.github/CODEOWNERS

```diff
@@ -968,6 +968,9 @@
 # PRLabel: %Spring App Discovery
 /sdk/springappdiscovery/Azure.ResourceManager.*/                   @sunkun99 @archerzz @ArcturusZhang @ArthurMa1978
 
+#PRLabel: %Quota
+/sdk/quota/Azure.ResourceManager.*/                                @tejasm-microsoft
```

Comment:
```suggestion
# PRLabel: %Quota
/sdk/quota/Azure.ResourceManager.*/                                @tejasm-microsoft

# ServiceLabel: %Quota %Mgmt
# ServiceOwners:                                                   @tejasm-microsoft
```

These need to follow the same format as the other management libraries.   Please move this block - with my suggestion - up to ~L959 to preserve alphabetical ordering by label.

Reply:
Followed the order suggested and was able to verify via PS command:
<img width="850" alt="image" src="https://github.com/user-attachments/assets/9f6caf9b-d888-4c93-86f7-78e4d357b419">